Packs/SekoiaXDR/IncidentTypes/Sekoia_XDR_Incident_Types.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "Sekoia XDR",
+    "version": -1,
+    "vcShouldIgnore": false,
+    "locked": false,
+    "name": "Sekoia XDR",
+    "prevName": "Sekoia XDR",
+    "color": "#CFD0D0",
+    "hours": 0,
+    "days": 0,
+    "weeks": 0,
+    "hoursR": 0,
+    "daysR": 0,
+    "weeksR": 0,
+    "system": false,
+    "readonly": false,
+    "default": false,
+    "autorun": true,
+    "closureScript": "CloseSekoiaAlert",
+    "disabled": false,
+    "reputationCalc": 2,
+    "onChangeRepAlg": 3,
+    "layout": "Sekoia XDR Layout",
+    "detached": false,
+    "extractSettings": {
+        "mode": "Specific",
+        "fieldCliNameToExtractSettings": {
+            "sekoiaalerturl": {
```

Comment:
didn't see that field - maybe we should change that to `externallink`?
Also, you can add the sourceIP filled ...

Reply:
Yes you are right, i should replace the `sekoiaalerturl`